src/tests/functional/plugin/shared/src/behavior/compiled_model/device.cpp

```diff
@@ -0,0 +1,28 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "behavior/compiled_model/device.hpp"
+
+namespace ov {
+namespace test {
+namespace behavior {
+
+INSTANTIATE_TEST_SUITE_P(OVCompiledModel, OVCompiledModelCorrectDevice, ::testing::Values("CPU", "GPU", "MULTI:CPU(4),GPU(4)"));
```

Comment:
Look like, a potential extension for `api conformance`. Could you please extend? Also, please remove test instance in shared lib